python/ray/dashboard/subprocesses/message.py

```diff
@@ -0,0 +1,79 @@
+from dataclasses import dataclass
+from typing import Union
+
+"""
+Child bound messages.
+"""
```

Comment:
Did you consider just using gRPC instead of implementing a custom RPC framework?